index.js

```diff
@@ -1,39 +1,342 @@
+          global.game.user.dropItem("seven days");
+        } else {
+          console.log("You don't have that item!");
+        }
+        global.game.user.showInventory();
+        global.game.currentRoom.showInventory();
+      } else if (input.includes("go") && input.includes("upstairs")) {
+        global.game.moveToRoom('Classroom');
+        inRoom = false;
+      } else if (input.includes("go") && input.includes("outside")){
+        global.game.moveToRoom('Main');
+        inRoom = false;
+        // catch-all for unrecognized input
+      } else {
+        console.log(`Sorry I don't know how tolet compareSimilarity = (input, triggerWords) => {
+${input}`);
+      }
+    }
+  } // end while
+  // go to the next room
+  play(global.game.currentRoom);
 }
+
+
+async function classroom() {
+  console.log(global.game.currentRoom.message);
+ 
+  let inRoom = true;
+
+  // write Foyer logic block
+  while (inRoom) {
+    console.log(`Player Status: ${global.game.user.checkStatus()}`);
+    let input = await ask('\n>_');
+    input = input.toLowerCase();
+
+    if (commonCommands(input)) {
+      // do nothing, because common command
+    } else {
+
+      // take the seven days paper
+      if ( input.includes("attend lecture") || input.includes("sit") ) {
+        console.log('You sit down in your normal seat. And Josh begins the lecture about recursion.');
+        global.game.user.changeStatus('Confused and Hungry');
+
+        // drop the seven days paper
+      } else if (input.includes("go") && input.includes("downstairs")) {
+        global.game.moveToRoom('Foyer');
+        inRoom = false;
+
+        // catch-all for unrecognized input
+      } else {
+        console.log(`Sorry I don't know how to ${input}`);
+      }
+    }
+  } // end while
+  // go to the next room
+  play(global.game.currentRoom);
+}
+
+/**** Stand alone objects ****/
+
+let roomNames = {
+  Main: "182 Main Street",
+  Foyer: "182 Main Street - Foyer",
+  Hallway: "Upstairs Hallway",
+  Classroom: "Second floor BCA Classroom"
+};
+
+let roomMessages = {
+  "182 Main Street":
+  `\n${roomNames.Main}:
+    You are standing on Main Street between Church and South Winooski.
+    There is a door here.A keypad sits on the handle.
+    On the door is a handwritten sign. `,
+
+  "182 Main Street - Foyer":
+  `\n${roomNames.Foyer}:
+    You are in a foyer. Or maybe it's an antechamber. Or a vestibule. 
+    Or an entryway. Or an atrium. Or a narthex. 
+    But let's forget all that fancy flatlander vocabulary, and just call it a foyer. In Vermont,
+    this is pronounced "FO-ee-yurr". A copy of Seven Days lies in a corner.`,
+
+  "Upstairs Hallway":
+  `\n${roomNames.Hallway }:
+    You are standing at the top of the stairs.`,
+
+  "Second floor BCA Classroom":
+    `\n${roomNames.Classroom}:
+      You are in the classroom. Josh is standing at the 
+      front waiting of the classroom`
+};
+
+let rooms = {
+  Main: new Room(roomNames.Main, ["welcome sign"]),
```

Comment:
Property names should be lowercase to conform to JavaScript naming conventions. Only class names should be capitalized. Your computer won't care, but anyone reading your code will.